src/config.rs

```diff
@@ -218,6 +218,25 @@ pub struct Database {
     /// various things in development.
     #[serde(default)]
     pub dangerously_recreate: bool,
+
+    // sqlite run on start commands
```

Comment:
Ideally, this value should be supported for other cases as well.
So maybe add a comment explaining what it does?
For example: `Execute query after initializing the DB`